stories/6-Tab.stories.tsx

```diff
@@ -16,8 +18,41 @@ const tabItems: TabItem[] = [
 
 storiesOf("Tab", module).add("Tab", () => (
   <Fragment>
-    <Tab items={tabItems}>
+    <p>Uncontrolled Tab</p>
+    <Tab
+      items={tabItems}
+      onTabChange={(index) => {
```

Comment:
It should always get the `activeTabIndex` prop when there is a `onTabChange` because that means we want to control it.

Reply:
I updated the TabProps to fix mistakes like that. When one of the controlled props (`onTabChange` or `activeTabIndex`) is provided other is required too, and `initialActiveTabIndex` shouldn't be defined.